graphql/coding/userBadges/updateUserCodingBadges.ts

```diff
@@ -0,0 +1,31 @@
+import { gql } from "@apollo/client";
+export type user_coding_badges_insert_input = {
```

Comment:
For some reason this type has to be this name otherwise I run into an error that says something like Type_name_selected is not user_coding_badges_insert_input.

Reply:
"user_coding_badges_insert_input" shouldn't be a type - it's part of the gql syntax for bulk inserting several values into a DB

I think you're getting a bit confused here - take a look at how skill ratings feature works and how multiple skill ratings are updated into the DB